### ISSUE TITLE
Error messages in gVim, not in Vim.

### ISSUE THREAD

**Issue by MeisterLLD** [junegunn/fzf#1873]
- [x] I have read through the manual page (`man fzf`)
- [x] I have the latest version of fzf
- [x] I have searched through the existing issues

## Info

- OS
    - [x] Linux
    - [ ] Mac OS X
    - [ ] Windows
    - [ ] Etc.
- Shell
    - [x] bash
    - [ ] zsh
    - [ ] fish
    
## Problem / Steps to reproduce
Hi ! I've been using fzf as a vim plugin for a while. Recently I've switched to gVim. It works flawlessly on vim but on gVim. Even though it does work, everytime I call :FZF or :History I get some error messages that I don't get with vim. They disappear after a few seconds (but the slow down is still annoying) and a slow down (even a few seconds after the message) and then it works fine.

I've searched existing issues, solutions were about updating but I do have last fzf version and Vim 8.2. 

Here are some screenshots, in French sorry.
https://imgur.com/gjuee5C
https://imgur.com/Rf0ljOy
Basically it says that it cannot modify something (about fzf.vim, strangely, which I guess should not be modified just by calling :FZF or :History), and a missing endif (only in :History)

**Comment by junegunn** [comment]
Is it reproducible with the minimal configuration?

- https://gist.githubusercontent.com/junegunn/6936bf79fedd3a079aeb1dd2f3c81ef5/raw

```sh
vim -Nu <(curl https://gist.githubusercontent.com/junegunn/6936bf79fedd3a079aeb1dd2f3c81ef5/raw)
```

**Comment by MeisterLLD** [comment]
Thanks for the answer. I just tried it : yes, exact same messages.

**Comment by junegunn** [comment]
What is `~/.vim/ftplugin/fzf.vim` file in the error message? fzf doesn't provide any `ftplugin` file, and according to the message, it looks like the file is not a proper VimScript file.

**Comment by MeisterLLD** [comment]
You are right, that is just it, thanks. I guess this file was some old
remainder of a failed Vundle installation a long time ago. I just deleted
it and everything works fine now. Sorry for the inconvenience.

Le sam. 15 févr. 2020 à 14:48, Junegunn Choi <notifications@github.com> a
écrit :
